apiml-security-common/build.gradle

```diff
@@ -1,29 +1,32 @@
 dependencies {
-    compile(project(':apiml-common'))
-    compile libraries.spring_boot_starter_web
-    compile libraries.spring_security_web
-    compile libraries.spring_security_config
-    compile libraries.apache_commons_lang3
-    compile libraries.bcprov;
-    compile libraries.bcpkix;
-    
+    implementation project(':apiml-common')
+    implementation project(':apiml-utility')
+    implementation project(':common-service-core')
+    implementation libraries.apache_commons_lang3
```

Comment:
I used the api configuration in dependencies for the 'common' projects to make use of transitive dependency for the majority of our subprojects. But I feel that the whole dependency tree will need to be revisited and analyzed more thoroughly. I was trying to follow advices in [this article](https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_recognizing_dependencies) but it would take me lot of time to analyze the code and identify properly which library should be api and which implementation.

Reply:
I was just talking about `implementation libraries.apache_commons_lang3`, it's here twice

Reply:
Ups, second one just removed. Good catch, thanks.